openassessment/fileupload/views.py

```diff
@@ -0,0 +1,163 @@
+import hashlib
```

Comment:
A general comment about this file: The views contained therein are supposed to support both backends encapsulated behind the `BaseBackend` interface. However, local filesystem backend code is contained in this view file - and it's unclear which code is relevant to which backend. Ideally the filesystem-specific methods should be moved behind the backend interface. At the very least, it should be clear which methods are relevant to which backend implementation - maybe in a separate file.